encoding/encodeInto.any.js

```diff
@@ -1,3 +1,21 @@
+function createBuffer(type, length = 0) {
+  if (type === "ArrayBuffer") {
+    return new ArrayBuffer(length);
+  } else {
+    // See https://github.com/whatwg/html/issues/5380 for why not `new SharedArrayBuffer()`
+    // WebAssembly.Memory's size is in multiples of 64 KiB
```

Comment:
Does this mean we actually create a 64 KiB buffer here?

Could we instead grab a copy of `buffer`'s constructor and use that to construct a SharedArrayBuffer instance of the exact side specified?

Reply:
Oh, that's cunning. Yeah, I think that would work.

cc @syg @domenic

Reply:
Done.